packages/babel-helpers/src/helpers/curryThis.ts

```diff
@@ -0,0 +1,7 @@
+/* @minVersion 7.24.0 */
+
+export default function _curryThis(fn: Function) {
+  return function (thisArg: any) {
+    return fn.apply(thisArg, [].slice.call(arguments, 1));
+  };
+}
```

Comment:
Equivalent to
```js
export default function _curryThis(fn: Function) {
  return function (thisArg: any) {
    return fn.call(...arguments);
  }
}
```
equivalent to
```js
export default function _curryThis(fn: Function) {
  return function (thisArg: any) {
    return Function.call.bind(fn)(...arguments);
  }
}
```
equivalent to
```js
export default function _curryThis(fn: Function) {
  var bound = Function.call.bind(fn);
  return function (thisArg: any) {
    return bound(...arguments);
  }
}
```
equivalent to
```js
export default function _curryThis(fn: Function) {
  return Function.call.bind(fn);
}
```
equivalent to
```js
export default function _curryThis(fn: Function) {
  return fn.call.bind(fn);
}
```
:)